pydoctor/templatewriter/__init__.py

```diff
@@ -27,7 +27,7 @@ def parse_xml(text: str) -> minidom.Document:
     Create a L{minidom} representaton of the XML string.
     """
     try:
-        return minidom.parseString(text)
+        return cast(minidom.Document, minidom.parseString(text))
```

Comment:
I agree with the cast as a temporary solution, but we should also submit a PR to `typeshed` to add a return type for `parseString()`.